tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -801,7 +801,7 @@ public async Task AcquireTokenWithAuthenticationCanceledTestAsync()
                         .ExecuteAsync(CancellationToken.None)
                         .ConfigureAwait(false);
                 }
-                catch (MsalClientException exc)
+                catch (MsalServiceException exc)
                 {
                     Assert.IsNotNull(exc);
                     Assert.AreEqual("authentication_canceled", exc.ErrorCode);

```

Comment:
>Assert.AreEqual("authentication_canceled", exc.ErrorCode); [](start = 20, length = 58)

I thought this was a client exception?

Reply:
I'm puzzled that there are not other tests that are failing as part of this change... 

---
In reply to: [309523923](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1305#discussion_r309523923) [](ancestors = 309523923)

Reply:
@henrik-me I'm puzzled too..only two tests failed as a result of the change. I'll try out the scenario above and report back tomorrow

Reply:
@henrik-me - take a look at this now. we now get a client exception, but not sure if this is the best way to handle it...also, before this change, were we doing service exceptions? i agree, does not seem right.

Reply:
Did you confirm that this is the actual user canceling/closing the dialog exception? If yes, then I believe the fix is right as it's ensures the abort is a client exception. Do we have any other scenarios that will cause this exception to be thrown? I'm wondering if any other scenario's exists that will go through this code path... 

---
In reply to: [309890518](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1305#discussion_r309890518) [](ancestors = 309890518)

Reply:
@henrik-me i tested that actually canceling/closing the dialog throws this exception, so anytime the user cancels, not sure how ests defines that, we will throw a client exception here.

Reply:
ESTS doesn't define that. we do. Which is why I'm curios if we will ever end in a situation where we will see a service exception from that piece of code. The service exceptions are thrown elsewhere. Anyway, I believe this fix is safe and I can't currently imagine there is a real breaking change anywhere.